R/validate.R

```diff
@@ -64,7 +64,7 @@ dm_validate <- function(x) {
     select(table = ref_table, column = ref_column) %>%
     check_colnames(dm_col_names, "Parent key")
 
-  stopifnot(lengths(def$pks) %in% 0:1)
+  stopifnot(lengths(def$pks) %in% 0:2)
```

Comment:
I don't see why this is necessary. Is the check a mistake?

Reply:
The definition of `new_pk()` is the following:

```r
new_pk <- function(column = list(), autoincrement = logical()) {
  stopifnot(is.list(column), is.logical(autoincrement))
  tibble(column = column, autoincrement = autoincrement)
}
```

This means that the length (`length(new_pk())`) can be `2L`.

Or are you saying that this entire check itself is unnecessary?

Reply:
I guess the check should be `NROW(new_pk()) %in% 0:1` .